pkg/pyinst/pyinst-build.mk

```diff
@@ -41,16 +44,30 @@ pyinst-distribution-data:
 	cp pkg/PixelatedWebmail.README $(DIST_VERSION)
 	cp LICENSE $(DIST_VERSION)
 
-pyinst-linux-helpers:
+pyinst-helpers-linux:
 	mkdir -p $(DIST_VERSION)apps/eip/files
 	# TODO compile static
 	cp /usr/sbin/openvpn $(DIST_VERSION)apps/eip/files/leap-openvpn
 	cp pkg/linux/bitmask-root $(DIST_VERSION)apps/eip/files/
 	cp pkg/linux/leap-install-helper.sh $(DIST_VERSION)apps/eip/files/
 	cp pkg/linux/polkit/se.leap.bitmask.bundle.policy $(DIST_VERSION)apps/eip/files/
 	mkdir -p $(DIST_VERSION)apps/mail
-	# TODO compile static
-	cp /usr/bin/gpg $(DIST_VERSION)apps/mail
+	cp ~/leap_thirdparty_build/gpg $(DIST_VERSION)apps/mail
```

Comment:
yes! messy stuff with having too many open branches. I think it'll be gone when I rebase to develop now.